packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -21,6 +21,7 @@ import { ReadableSpan, SpanExporter } from '@opentelemetry/tracing';
 import { Socket } from 'dgram';
 import { spanToThrift } from './transform';
 import * as jaegerTypes from './types';
+import { OT_REQUEST_HEADER } from './utils';
```

Comment:
what is this for?

Reply:
The explanation can be read at: https://github.com/open-telemetry/opentelemetry-js/pull/965#discussion_r407751053.

I followed the zipkin exporter folder pattern.

Reply:
The explanation can be read at: https://github.com/open-telemetry/opentelemetry-js/pull/965#discussion_r407751053.

I followed the zipkin exporter folder pattern.